helm/private/helm_package.bzl

```diff
@@ -199,16 +156,14 @@ def _helm_package_impl(ctx):
         ),
     ]
 
-helm_package = rule(
+_helm_package = rule(
```

Comment:
One thing I dislike about the private rule definitions is they can make queries more sensitive
```
(.venv) ~/Code/rules_helm (dev/chart-cleanup ✔) bazel query 'kind("^helm_package", //...)'
//tests/with_image_deps:with_image_deps_regex_test
Loading: 0 packages loaded
(.venv) ~/Code/rules_helm (dev/chart-cleanup ✔) bazel query 'kind("^_helm_package", //...)'
//tests/inline_chart:inline_chart
//tests/inline_values:inline_values
//tests/simple:simple
//tests/version_stamp:version_stamp.no_stamp
//tests/version_stamp:version_stamp.stamp
//tests/with_chart_deps:with_chart_deps
//tests/with_image_deps:with_image_deps
```
Maybe there can be a `helm_package_interface.bzl` so the generator function and rule can share the same name?

Reply:
Closing in favor of https://github.com/abrisco/rules_helm/pull/47#discussion_r1466598300